ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorUtil.java

```diff
@@ -85,6 +86,23 @@
 public class CompactorUtil {
   private static final Logger LOG = LoggerFactory.getLogger(CompactorUtil.class);
   public static final String COMPACTOR = "compactor";
+  public static final String CompactionLevelConf = "hive.compaction.level";
```

Comment:
why not use COMPACTOR constant?

Reply:
As discussed, reverted back to using RewritePolicy.PARTITION